src/condor_ce_run

```diff
@@ -286,7 +286,7 @@ def main():
 if __name__ == '__main__':
     try:
         sys.exit(main())
-    except ce.CondorRunException, e:
+    except ce.CondorRunException as e:
```

Comment:
```suggestion
    except ce.CondorRunException as exc:
```

GitHub's not letting me create a large enough suggestion so you'll also need to update line 294